amis/build_ami.sh

```diff
@@ -9,6 +9,7 @@ build_date=$4
 
 available_os="centos6 centos7 alinux ubuntu1404 ubuntu1604"
 available_regions="eu-west-1,eu-west-2,ap-southeast-1,ap-southeast-2,eu-central-1,ap-northeast-1,ap-northeast-2,us-west-2,sa-east-1,us-west-1,us-east-2,ap-south-1,ca-central-1"
+cwd="$(dirname $0)"
 
```

Comment:
What about defining an var  `BUILD_PATH="${cwd}/../.."` to be used in the code below and inside the json templates. 

Within a json template we can map the variable
```
{
 "variables" : {
     [...]
     "build_path" : "{{env `BUILD_PATH`}}",
 }
[...]
```
and then use `{{build_path}}/vendor/cookbooks`  instead of `{{template_dir}}/../../vendor/cookbooks`